src/main/java/edu/unc/lib/boxc/migration/cdm/status/SourceFilesSummaryService.java

```diff
@@ -95,68 +85,57 @@ public int totalFilesInProject() {
     /**
      * @return load list of all CSV records
      */
-    public List<CSVRecord> loadAllEntries(Path mappingPath) {
-        List<CSVRecord> allEntries = new ArrayList<>();
-
-        try (
-                Reader reader = Files.newBufferedReader(mappingPath);
-                CSVParser csvParser = new CSVParser(reader, CSVFormat.DEFAULT
-                        .withFirstRecordAsHeader()
-                        .withHeader(SourceFilesInfo.CSV_HEADERS)
-                        .withTrim());
-        ) {
+    public List<CSVRecord> loadAllFiles(Path mappingPath) {
+        List<CSVRecord> allFiles = new ArrayList<>();
+
+        try (var csvParser = SourceFileService.openMappingsParser(mappingPath)) {
             for (CSVRecord csvRecord : csvParser) {
-                allEntries.add(csvRecord);
+                allFiles.add(csvRecord);
             }
         } catch (IOException e) {
             log.error("Failed to list files", e);
         }
-        return allEntries;
+        return allFiles;
     }
 
     /**
      * @return load list of CSV records with source file populated
      */
-    public List<CSVRecord> loadPopulatedEntries(Path mappingPath) {
-        List<CSVRecord> populatedEntries = loadAllEntries(mappingPath);
+    public List<CSVRecord> loadPopulatedFiles(Path mappingPath) {
+        List<CSVRecord> populatedFiles = loadAllFiles(mappingPath);
 
-        return populatedEntries.stream().filter(entry -> !entry.get(SOURCE_FILE_FIELD).isEmpty())
+        return populatedFiles.stream().filter(entry -> !entry.get(SourceFilesInfo.SOURCE_FILE_FIELD).isEmpty())
                 .collect(Collectors.toList());
     }
 
     /**
      * @return list of new CSV records with source file populated
      */
-    public List<CSVRecord> getListNewEntries() {
-        List<CSVRecord> previousMappings = new ArrayList<>();
-        if (previousStatePopulatedFiles != null) {
-            previousMappings = previousStatePopulatedFiles;
-        }
-        List<CSVRecord> newMappings = loadPopulatedEntries(getNewMappingPath());
-
-        if (!previousMappings.isEmpty()) {
-            Set<String> previousIdentifiers = getIdentifierSet(previousMappings);
-            Set<String> newIdentifiers = getIdentifierSet(newMappings);
-            newIdentifiers.removeAll(previousIdentifiers);
-            return newMappings.stream().filter(entry -> newIdentifiers.contains(entry.get(ID_FIELD)))
-                    .collect(Collectors.toList());
-        } else {
+    public List<CSVRecord> getListNewPopulatedFiles() {
```

Comment:
how about `listNewlyPopulatedFiles`? My brain has a hard time parsing`getListNew`. And it might also help to update the comment on the method to something like "@return list of CSV records that have been populated since the previous mapping state"